src/goto-symex/symex_main.cpp

```diff
@@ -130,6 +133,26 @@ void goto_symext::symex_assume(statet &state, const exprt &cond)
   simplified_cond = state.rename(std::move(simplified_cond), ns).get();
   do_simplify(simplified_cond);
 
+  // It would be better to call try_filter_value_sets after apply_condition,
```

Comment:
Rather than dup this entire comment note this in brief and refer the reader to the other function